react/components/Table/SimpleTable.js

```diff
@@ -58,6 +61,33 @@ class SimpleTable extends Component {
       : DEFAULT_COLUMN_WIDTH
   }
 
+  addLineActionsToSchema(schema, lineActions) {
+    return (schema.properties = {
```

Comment:
this is a little weird. You could return a new schema since you receive the schema as param.
```js
return {
  properties: {
     ...schema.properties,
     .......here your code like it is...
  }
}
```